### PR TITLE
modify template operator readme

### PR DESCRIPTION
**Pull request by ruanxin**

### CODE REVIEW

**Review by adityabhatia — CHANGES_REQUESTED** [review]
changing the operator directory to `test-operator` will fail the `Makefile` commands. 

I would not rename it in this case.

File: samples/template-operator/README.md [hunk 1]
```diff
@@ -37,19 +37,19 @@ Additionally, it hides Kubernetes boilerplate code to develop fast and efficient
 
 ### Generate kubebuilder operator 
 
-1. Create a folder and make a directory, e.g. `operator`.
+1. Create a folder and make a directory, e.g. `test-operator`.
```
Comment (ruanxin) [inline samples/template-operator/README.md:40]: @adityabhatia, do you mean this `operator` is the operator folder under root folder? like `lifecycle-manager/operator`? Then I think the first sentence is misleading, to me, it' means the root folder name. 
```suggestion
1. Create module folder, e.g. `test-operator` and make a directory `operator`
```

File: samples/template-operator/README.md [hunk 2]
```diff
@@ -37,19 +37,19 @@ Additionally, it hides Kubernetes boilerplate code to develop fast and efficient
...
     ```sh
-    mkdir operator && cd operator
+    mkdir test-operator && cd test-operator
```
Comment (ruanxin) [inline samples/template-operator/README.md:42]:
```suggestion
    mkdir operator && cd operator
```

File: samples/template-operator/README.md (hunk 1 above)
Comment (adityabhatia) [reply samples/template-operator/README.md:40]: Yes that's right. Check [Makefile](https://github.com/kyma-project/lifecycle-manager/blob/259a8216545a7d7aec8dd8d6ce5440db497da55f/samples/template-operator/Makefile) 
and look for `operator/..`

_"Create `module` folder"_ is confusing. I would suggest _"In your root directory"_ to be more generic.

File: samples/template-operator/README.md [hunk 3]
```diff
@@ -37,19 +37,19 @@ Additionally, it hides Kubernetes boilerplate code to develop fast and efficient
...
+1. In your project (module operator) root folder make a directory `operator`
     ```sh
     mkdir operator && cd operator
     ```
 
-2. Initialize `kubebuilder` project. Please make sure domain is set to `operator.kyma-project.io`.
+2. Initialize `kubebuilder` project. Please make sure domain is set to `kyma-project.io`.
     ```sh 
-   kubebuilder init --domain operator.kyma-project.io --repo github.com/kyma-project/test-operator --plugins=go/v4-alpha
+   kubebuilder init --domain kyma-project.io --repo github.com/kyma-project/test-operator/operator --plugins=go/v4-alpha
```
Comment (adityabhatia) [inline samples/template-operator/README.md]:
```suggestion
   kubebuilder init --domain kyma-project.io --repo github.com/kyma-project/<your-repo-name> --plugins=go/v4-alpha
```

**Review by adityabhatia — APPROVED** [review]

**Pull request merged**